test/json_api/assert_payload_test.exs

```diff
@@ -909,4 +909,164 @@ defmodule Voorhees.Test.JSONApi.AssertPayloadTest do
       })
     end
   end
+
+  test "should pass with equal payloads containing both data and included" do
+    payload = %{
+      "data" => %{
+        "type" => "user",
+        "id" => "1",
+        "attributes" => %{
+          "email" => "test@example.com",
+          "name" => "Tester"
+        }
+      },
+      "included" => [%{
+        "id" => "1",
+        "type" => "user",
+        "attributes" => %{
+          "name" => "Tester1"
+        }
+      }]
+    }
+
+    Voorhees.JSONApi.assert_payload payload, %{
+      "data" => %{
+        "type" => "user",
+        "id" => "1",
+        "attributes" => %{
+          "email" => "test@example.com",
+          "name" => "Tester"
+        }
+      },
+      "included" => [%{
+        id: "1",
+        type: "user",
+        attributes: %{
+          name: "Tester1"
+        }
+      }]
+    }
+  end
+
+  test "throw an error when payload does not contain data or included, but is expected" do
+    payload = %{}
+
+    assert_raise ExUnit.AssertionError, """
+    Payload did not match expected
+
+    "data" did not match expected
```

Comment:
Would rather have something that says `"data" was expected, but was not present`